.github/workflows/release.yaml

```diff
@@ -10,7 +10,8 @@ jobs:
   build:
     runs-on: ubuntu-latest
     env:
-      image_name: docker.io/bitnami/sealed-secrets-controller
+      dockerhub_image_name: docker.io/bitnami/sealed-secrets-controller
```

Comment:
the repository variable value is bitnami-labs/sealed-secret-controller and we are using right now bitnami/sealed-secret-controller. We are not align with the names in dockerhub

Reply:
Because it's not the same. The repository is `bitnami-labs/sealed-secrets` and not `bitnami/sealed-secrets`.

My 2 cents, just follow the DockerHub approach and hardcode it too in ghcr.

Reply:
Done